packages/kbn-config/src/deprecation/deprecation_factory.test.ts

```diff
@@ -103,7 +104,8 @@ describe('DeprecationFactory', () => {
                   "Replace \\"myplugin.oldsection.deprecated\\" with \\"myplugin.newsection.renamed\\" in the Kibana config file, CLI flag, or environment variable (in Docker only).",
                 ],
               },
-              "message": "\\"myplugin.oldsection.deprecated\\" is deprecated and has been replaced by \\"myplugin.newsection.renamed\\"",
+              "message": "Setting \\"myplugin.oldsection.deprecated\\" is deprecated and has been replaced by \\"myplugin.newsection.renamed\\"",
```

Comment:
I'd avoid repeating the title in the message. 

```suggestion
              "message": "Use  \\"myplugin.newsection.renamed\\" instead.",
```